pwa-shell/src/ui/menu/Logout.js

```diff
@@ -1,22 +1,36 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import { defineMessages } from 'react-intl.macro';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { FormattedMessage } from 'react-intl.macro';
 
 import { ReactComponent as LogoutIcon } from 'images/icons/ic_exit.svg';
 
+const messages = defineMessages({
+  noname: {
+    id: 'profile.noname',
+    defaultMessage: 'Registered User',
+  },
+});
+
 class Logout extends Component {
+  componentWillMount() {
```

Comment:
`componentWillMount` is deprecated. Please use `componentDidMount`.

Reply:
ugh, sorry about that. method changed.